lib/identity_config.rb

```diff
@@ -28,7 +30,7 @@ class << self
     end,
     symbol: proc { |value| value.to_sym },
     comma_separated_string_list: proc do |value|
-      value.split(',')
+      value.parse_csv.to_a
```

Comment:
personal style: I would prefer the explicit `CSV.parse_line` over the monkeypatch, makes it clear it will only return a single row (instead of array of arrays from full CSV parseing)
```suggestion
      CSV.parse_line(value).to_a
```

Reply:
Yes, `String#parse_csv` just calls to `CSV.parse_line`, I was saying that the latter has a clearer name to me.

CSV gem source:
https://github.com/ruby/csv/blob/b9c80bd5c37f25b9ca7d5b7e6a875bada0f8bc0f/lib/csv/core_ext/string.rb#L6-L8

Reply:
Gotcha, I can make a quick follow-on

Reply:
Updated in https://github.com/18F/identity-idp/pull/10358